go.mod

```diff
@@ -73,6 +72,8 @@ require (
 	sigs.k8s.io/e2e-framework v0.3.0
 )
 
+require github.com/openshift/osde2e-common v0.0.0-20240122091653-ac2a77b4ad4d
```

Comment:
/hold

you should bump osde2e-common by doing a `go get github.com/openshift/osde2e-common@latest` then a `go mod tidy`